http/reactive-routes/src/test/java/io/quarkus/ts/http/reactiveroutes/validation/ValidationOnResponseRouteHandlerIT.java

```diff
@@ -39,18 +38,4 @@ public void shouldGetValidationErrorWhenUniResponseIdIsWrong() {
         assertValidationErrorField(response, "id", "id can't be null");
     }
 
-    @Disabled("Not validating Java types. Reported in: https://github.com/quarkusio/quarkus/issues/15168")
```

Comment:
Please just double-check that we need to return an object and then the @valid annotation works.

Reply:
According to Clement1], we do not need to do this for reactive routes. Instead, the way forward is Resteasy-reactive and required test is added in this PR

[1] https://github.com/quarkusio/quarkus/issues/15168#issuecomment-922661864

Reply:
ok